Dockerfile-developBranch

```diff
@@ -9,7 +9,7 @@ WORKDIR /code
 RUN pip3 install --upgrade pip
 RUN pip3 install --requirement requirements.txt
 
-CMD [ "rq", "worker", "--config", "rq_settings", "--name", "D43_Dev_JobHandler" ]
```

Comment:
I'm not sure if this should be `D43_Dev_CatalogJobHandler` or `D43Catalog_Dev_JobHandler`.
This question applies to the `Dockerfile-masterBranch` file as well.

Reply:
It's just how the door43-enqueue-job repo will add the job to the stack, so whatever you want. D43_Dev_CatalogJobHandler should be fine. This is of course for development environment.